models/user.rb

```diff
@@ -0,0 +1,83 @@
+class User < ActiveRecord::Base
+  has_many :study_assignments
+
+  def self.all_inactive
+    Users.all.reject do |user|
+      user.study_assignments.any? { |assignment| assignment.is_active? }
+    end
+  end
+
+  def current_assignments
+    study_assignments.undeleted
+  end
+
+  def assign_to_study(study_uuid, role, sites = [])
+    assignment = study_assignments.find { |assignment| assignment.study_uuid == study_uuid }
+
+    if assignment
+      if role.is_a?(Array)
```

Comment:
The parameter name is misleading as it is just "role" and could be either role or roles. Additionally, the type checking done in this if statement to see if it is an array of roles or a singular role is confusing as well. This can potentially be separated into 2 methods for either role or roles.